filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilder.scala

```diff
@@ -0,0 +1,60 @@
+package cromwell.filesystems.blob
+
+import cromwell.core.path.{NioPath, PathBuilder}
+import scala.util.Try
+import java.nio.file.FileSystems
+import java.net.URI
+import scala.language.postfixOps
+import com.google.common.net.UrlEscapers
+import com.azure.storage.blob.nio.AzureFileSystem
+import scala.jdk.CollectionConverters._
+import cromwell.core.path.Path
+import scala.util.Failure
+import com.azure.core.credential.AzureSasCredential
+import cromwell.filesystems.blob.BlobPathBuilder._
```

Comment:
We typically keep the imports sorted. This might be something that IntelliJ does automatically... It looks like we're not super consistent about where we insert blank lines, but within each stanza they should be sorted lexicographically. 

FWIW my IntelliJ is configured to do:
``` 
base package imports 
--blank line--
all other imports 
--blank line--
java
javax
scala
```

Reply:
Got it, I think thats the default for VSCode as well, but it apparently doesn't run on file write